src/core/Akka.Remote/FailureDetector.cs

```diff
@@ -34,12 +35,11 @@ public abstract class FailureDetector
 
         #region Static members
 
-        //fixed: sign will no longer flip, but the tickcount will go back down to zero every 24.9 days 
         /// <summary>
-        /// TBD
+        /// The default clock implementation used by the <see cref="PhiAccrualFailureDetector"/>
         /// </summary>
-        /// <returns>TBD</returns>
-        public static readonly Clock DefaultClock = () => Environment.TickCount & Int32.MaxValue;
+        /// <returns>A clock instance.</returns>
+        public static readonly Clock DefaultClock = () => MonotonicClock.GetTicks();
```

Comment:
Key fix to the issue here. Switched to 64 bit timer instead of the previous 32bit implementation with `Environment.TickCount`.